src/game_state/Player.h

```diff
@@ -17,16 +17,20 @@ class Player : public GameObject {
   Player();
   ~Player();
 
+  void Reset();
   float GetVerticalVelocity();
+  float GetZVelocity();
   bool GetSpacebarDown();
   std::shared_ptr<GameObject> GetGround(); // null if no ground
   std::shared_ptr<Shape> GetModel() override;
+  ObjectType GetType() override;
 
   void SetVerticalVelocity(float vertical_velocity);
```

Comment:
Since you added ZVelocity, you should change the naming of VerticalVelocity to YVelocity in the getter and setter to make it consistent